server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/BitcoinExtension.java

```diff
@@ -59,7 +60,8 @@ public IExchange createExchange(String paramString) //(Bitstamp is in built-in e
                 String apiKey = paramTokenizer.nextToken();
                 String apiSecret = paramTokenizer.nextToken();
                 return new BitfinexExchange(apiKey, apiSecret);
-            } if ("bittrex".equalsIgnoreCase(prefix)) {
+            }
+            else if ("bittrex".equalsIgnoreCase(prefix)) {
```

Comment:
Great you spotted the missing `else`, but please keep the code style consistent. `else if` should start on the same line the previous `if`-block ends on.